coral-common/src/main/java/com/linkedin/coral/common/functions/FunctionReturnTypes.java

```diff
@@ -19,11 +19,10 @@
 import com.linkedin.coral.com.google.common.collect.ImmutableList;
 
 
-public class HiveReturnTypes {
-
-  private HiveReturnTypes() {
-
-  }
+/**
+ * This class provides function return types that are not in {@link ReturnTypes}
+ */
+public class FunctionReturnTypes {
```

Comment:
do we want to preserve the private constructor ?  
I assume the reason this change is not breaking compatibility is due to the private constructor, isn't it ?  If so let's preserve it ... 

And this is technically not 100% safe since there could be extension for this class in other code base. So maybe make it a final class?

Reply:
Agreed, added it back